services/app/src/components/views/Settings.tsx

```diff
@@ -55,8 +55,8 @@ const Settings = (props: Props): JSX.Element => {
       <Button className="primary large" onClick={() => props.onExpansionSelect()}>Choose game / expansion</Button>
       <p className="expansionLabel">Currently playing: <strong>Expedition {props.contentSets.horror ? <span> + Horror</span> : null}{props.contentSets.future ? <span> + Future</span> : null}</strong></p>
 
-      <Picker label="Adventurers" value={props.numPlayers} onDelta={(i: number) => props.onPlayerDelta(props.numPlayers, i)}>
-        {(props.numPlayers > 1) ? 'The number of players.' : <div><strong>Solo play:</strong> Play as two adventurers with double the combat timer.</div>}
+      <Picker label="Adventurers" value={props.numLocalPlayers} onDelta={(i: number) => props.onPlayerDelta(props.numLocalPlayers, i)}>
```

Comment:
Should probs also indicate the total sum of multiplayer players here

Reply:
(see above comment on dialogs)